src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Lists/BitBasicListTests.cs

```diff
@@ -0,0 +1,121 @@
+                parameters.Add(p => p.Items, GetTestData(itemCount));
+                parameters.Add(p => p.ListStyle, (listHeight is null) ? "height:500px" : $"height:{listHeight}px");
+                parameters.Add(p => p.ItemStyle, (itemHeight is null) ? "height:50px" : $"height:{itemHeight}px");
+                //ItemSize default value is 50.
+                parameters.Add(p => p.ItemSize, (itemSize is null) ? 50 : itemSize.Value);
+                //OverscanCount default value is 3.
+                parameters.Add(p => p.OverscanCount, (overscanCount is null) ? 3 : overscanCount.Value);
+            });
+
+            var bitList = component.Find(".bit-bsc-lst");
+
+            if (virtualize)
+            {
+                listHeight = (listHeight is null) ? 500 : listHeight;
+                itemHeight = (itemHeight is null) ? 50 : itemHeight;
+                overscanCount = (overscanCount is null) ? 3 : overscanCount;
+
+                //When virtualize is true, number of rendered items is greater than number of items showm in the list + 2 * overScanCount.
+                var expectedRenderedItemCount = Math.Ceiling((decimal)(listHeight / itemHeight)) + 2 * overscanCount;
+                var actualRenderedItemCount = bitList.GetElementsByClassName("list-item").Length;
+                Assert.IsTrue(actualRenderedItemCount >= expectedRenderedItemCount);
+            }
+            else
+            {
+                var actualRenderedItemCount = bitList.GetElementsByClassName("list-item").Length;
+                Assert.AreEqual(itemCount, actualRenderedItemCount);
+            }
+        }
+
+        [DataTestMethod,
+            DataRow(100, "AssignedRole"),
+            DataRow(100, null)
+        ]
+        public void BitBasicListShouldHaveCorrectRole(int itemCount, string role)
+        {
+            var component = RenderComponent<BitBasicListTest>(parameters =>
+            {
+                parameters.Add(p => p.Items, GetTestData(itemCount));
+                if (!String.IsNullOrEmpty(role))
+                    parameters.Add(p => p.Role, role);
+            });
+
+            var bitList = component.Find(".bit-bsc-lst");
+
+            var bitLisRole = bitList.GetAttribute("role");
+
+            if (String.IsNullOrEmpty(role))
+            {
+                Assert.AreEqual("list", bitLisRole);
+            }
+            else
+            {
+                Assert.AreEqual(role, bitLisRole);
+            }
+        }
+
+        [DataTestMethod, DataRow(100)]
+        public void BitBasicListShouldHaveCorrectClass(int itemCount)
+        {
+            var component = RenderComponent<BitBasicListTest>(parameters =>
+            {
+                parameters.Add(p => p.Visual, Visual.Fluent);
+                parameters.Add(p => p.Items, GetTestData(itemCount));
+            });
+
+            var bitList = component.Find(".bit-bsc-lst");
+
+            Assert.IsTrue(bitList.ClassList.Contains("bit-bsc-lst"));
```

Comment:
this assertion is meaningless :)

Reply:
It's because of the general class name of BitBasicList :)